test_common/harness/msvc9.c

```diff
@@ -714,6 +714,7 @@ int32_t float2int(float fx)
     return u.i;
 }
 
+#if !__has_builtin(__builtin_clz)
```

Comment:
Two small questions here:

1. Do we need any specific compiler checks for `__has_builtin`?
2. Would this be more readable with the two conditions on the same line:
`#if !defined(_WIN64) && !__has_builtin(__builtin_clz)`

Reply:
1. Yes, we need checks for `__has_builtin`. Updated.
2. I don't combine the 2 conditions together as `#if !defined(_WIN64) && !__has_builtin(__builtin_clz)` because that will require the `#else` (corresponding to `#if !defined(_WIN64)`) also need a condition `#if !__has_builtin(__builtin_clz)`.